Cargo.toml

```diff
@@ -16,11 +19,13 @@ features = ["stm32f303xc", "rt", "stm32-usbd"]
 targets = ["thumbv7em-none-eabihf"]
 
 [dependencies]
+cfg-if = "0.1"
 cortex-m = "0.6"
 cortex-m-rt = "0.6"
 embedded-hal = "0.2"
 nb = "0.1"
-stm32f3 = "0.11"
+paste = "1"
+stm32f3 = { git = "https://github.com/stm32-rs/stm32-rs-nightlies" }
```

Comment:
We'll need a new release of the stm32f3 crate before we can merge.

Reply:
Thanks for your PR there ;)